src/core/config/ascending_od/option.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "config/ascending_od/type.h"
+#include "config/common_option.h"
+
+namespace config {
+extern CommonOption<AscendingODFlagType> const kAscendingODOpt;
```

Comment:
'AscendingOD' option is useful only for OD verifier and won't be needed by other algorithms. So I think that it should be removed. Instead, you can use `Option` as shown [here](https://github.com/Desbordante/desbordante-core/blob/main/src/core/algorithms/ind/faida/faida.cpp#L18).